lib/locales/en.ts

```diff
@@ -227,7 +231,9 @@ export const locale: Locale = {
     assumption_of_the_blessed_virgin_mary: 'The Assumption of the Blessed Virgin Mary',
     athanasius_of_alexandria_bishop: 'Saint Athanasius, Bishop and Doctor of the Church',
     attracta_of_killaraght_virgin: 'Saint Attracta, Virgin',
+    aubert_of_avranches_bishop: 'Saint Aubert, Bishop',
     audoen_of_rouen_bishop: 'Saint Audoen, Bishop',
+    auguste_chapdelaine_priest: 'Saint Auguste Chapdelaine, Priest and Martyr',
```

Comment:
Somewhat I’d prefer using `Augustus` instead of `Auguste` (cf [this](https://www.catholic.org/saints/saint.php?saint_id=1613)).

```suggestion
    augustus_chapdelaine_priest: 'Saint Augustus Chapdelaine, Priest and Martyr',
```